src/client/src/components/DropdownMenu.stories.tsx

```diff
@@ -9,14 +9,12 @@ export default {
     options: ["George", "Paul", "Ringo", "John"],
     onSelectionChange: console.log,
   },
-} as ComponentMeta<typeof DropdownMenu>
+} as Meta<typeof DropdownMenu>
 
-const Template: ComponentStory<typeof DropdownMenu> = (args) => (
-  <DropdownMenu {...args} />
-)
+export const WithoutSelectionSpecified = {
+  args: {},
+}
 
-export const WithoutSelectionSpecified = Template.bind({})
-WithoutSelectionSpecified.args = {}
-
-export const WithSelectionSpecified = Template.bind({})
-WithSelectionSpecified.args = { selectedOption: "Ringo" }
+export const WithSelectionSpecified = {
+  args: { selectedOption: "Ringo" },
```

Comment:
obviously written by someone with a penchant for drummers ...

Reply:
At least I know who this is 🤣